### PR TITLE
Make page connect between main page and dashboard page

### PR DESCRIPTION
**Pull request by HaiyueZhang**
When click and login in successfully, we will enter the dashboard page.

When we click "Parachute" in Navbar, we will back to the main page.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -35,10 +37,14 @@ const Home: NextPage = () => {
...
-            <button onClick={() => void signIn("google")} className="bg-black text-white text-center font-semibold rounded-lg w-full p-3">
+            <button onClick={() => void signIn("google", { 
+                          callbackUrl: `${window.location.origin}/dashboard`,
+                  })} className="bg-black text-white text-center font-semibold rounded-lg w-full p-3">
               Sign in with Google
             </button>
-            <button onClick={() => void signIn("auth0")} className="border text-center font-semibold rounded-lg w-full p-3">
+            <button onClick={() => void signIn("auth0", { 
+                          callbackUrl: `${window.location.origin}/dashboard`,
+                  })} className="border text-center font-semibold rounded-lg w-full p-3">
```
Comment (louisunlimited) [inline src/pages/index.tsx:47]: Good work on this! However, we should enforce redirect from `/` base URL to `/dashboard` if we detech user login. This way if they clicked `parachute.fyi` via a web link they'll still get redirected to `/dashboard`

**Review by louisunlimited — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (RaymondWHZ) [reply src/pages/index.tsx:47]: We may implement this after a while. #24

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,10 +1,89 @@
...
+            <Popover.Button className="focus:outline-none">
+              <Image
+                className="cursor-pointer rounded-full"
+                src={image}
+                alt="User dropdown"
+                width={40}
+                height={40}
+              />
+            </Popover.Button>
+            <Popover.Panel>
```
Comment (louisunlimited) [inline src/components/Navbar.tsx:43]: Can we make this popover `onHover` instead of `onClick`?

**Review by louisunlimited — CHANGES_REQUESTED** [review]
Great work! Can you also handel session check on `/dashboard` as well?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -2,10 +2,12 @@ import { type NextPage } from "next";
...
 import { useElementMouseRelativeAngle } from "../utils/hooks";
-import { signIn, signOut } from "next-auth/react";
+import { signIn} from "next-auth/react";
+// import { useRouter } from 'next/router';
 
 const Home: NextPage = () => {
   const { angle, ref } = useElementMouseRelativeAngle();
+  // const router = useRouter();
   return (
     <>
```
Comment (louisunlimited) [inline src/pages/index.tsx:12]: I was thinking about resolving #24 right now by checking session and routerpush /dashboard

**Review by louisunlimited — CHANGES_REQUESTED** [review]

**Review by RaymondWHZ — APPROVED** [review]
Great work!

**Pull request merged**